generators/app/index.js

```diff
@@ -0,0 +1,502 @@
+      this.destinationPath("src/index.js"),
+      {}
+    );
+
+    mkdirp(this.destinationPath("src/app/assets/"));
+
+    this.fs.copyTpl(
+      this.templatePath(
+        "src/app/components/Routes/components/AuthenticatedRoute.js"
+      ),
+      this.destinationPath(
+        "src/app/components/Routes/components/AuthenticatedRoute.js"
+      ),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/app/components/Routes/constants.js"),
+      this.destinationPath("src/app/components/Routes/constants.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/app/components/Routes/index.js"),
+      this.destinationPath("src/app/components/Routes/index.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/app/components/Routes/styles.js"),
+      this.destinationPath("src/app/components/Routes/styles.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/app/index.js"),
+      this.destinationPath("src/app/index.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/app/styles.js"),
+      this.destinationPath("src/app/styles.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/config/api.js"),
+      this.destinationPath("src/config/api.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/config/i18n.js"),
+      this.destinationPath("src/config/i18n.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/config/perf.js"),
+      this.destinationPath("src/config/perf.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/redux"),
+      this.destinationPath("src/redux"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/redux/store.js"),
+      this.destinationPath("src/redux/store.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/redux/Auth/actions.js"),
+      this.destinationPath("src/redux/Auth/actions.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/redux/Auth/reducer.js"),
+      this.destinationPath("src/redux/Auth/reducer.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/services"),
+      this.destinationPath("src/services"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/utils"),
+      this.destinationPath("src/utils"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/utils/colors.js"),
+      this.destinationPath("src/utils/colors.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("src/utils/array.js"),
+      this.destinationPath("src/utils/array.js"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("gitignore"),
+      this.destinationPath(".gitignore"),
+      {}
+    );
+    this.fs.copyTpl(
+      this.templatePath("eslintrc.js"),
+      this.destinationPath(".eslintrc.js"),
+      {}
+    );
+
+    this.fs.copyTpl(
+      this.templatePath("public/index.html"),
```

Comment:
We should be consistent with the naming of the templates. The template of `package.json` starts with `_`.  Use the same convention in all the files